src/gnatss/loaders.py

```diff
@@ -440,3 +447,56 @@ def load_quality_control(qc_files: List[str], time_scale="tt") -> pd.DataFrame:
         qc_df = pd.DataFrame(columns=[constants.QC_STARTTIME, constants.QC_ENDTIME])
 
     return qc_df
+
+
+def read_raw_l1_data_files(
+    data_files: list[str], data_format: Literal["INSPVAA", "INSSTDEVA"] = "INSPVAA"
+) -> pd.DataFrame:
+    """
+    Read from L1 data files and return its dataframe representation
```

Comment:
Please add links to the Novatel website
INSSTDEVA: https://docs.novatel.com/OEM7/Content/SPAN_Logs/INSSTDEV.htm?tocpath=Commands%20%2526%20Logs%7CLogs%7CSPAN%20Logs%7C_____30
INSPVAA: https://docs.novatel.com/OEM7/Content/SPAN_Logs/INSPVA.htm?tocpath=Commands%20%2526%20Logs%7CLogs%7CSPAN%20Logs%7C_____22